connectors/standalone/src/main/scala/io/delta/standalone/internal/OptimisticTransactionImpl.scala

```diff
@@ -408,8 +409,12 @@ private[internal] class OptimisticTransactionImpl(
    * Looks at actions that have happened since the txn started and checks for logical
    * conflicts with the read/writes. If no conflicts are found return the commit version to attempt
    * next.
+   *
+   * @throws DeltaConcurrentModificationException if the next commit version cannot be resolved
+   *                                              due to the concurrent modifications in the
```

Comment:
nit: s/modifications in/modifications to

Reply:
Thank you :+1: Fixed in https://github.com/delta-io/delta/pull/1947/commits/24e6e5cb3d6639b47c6cafdc1dfe528560cac001